app/src/docs/tables/Advanced.doc.tsx

```diff
@@ -12,13 +12,14 @@ export class AdvancedTablesDoc extends BaseDocsBlock {
 
                 <DocExample cx={ css.appBg } title="Columns Configuration" path="./_examples/tables/ColumnsConfig.example.tsx" />
 
-                <DocExample cx={ css.appBg } title="Table with rows Drag&Drop" path="./_examples/tables/TableWithDnD.example.tsx" />
+                <DocExample title="Table with rows Drag&Drop" path="./_examples/tables/TableWithDnD.example.tsx" />
```

Comment:
Add cx={ css.appBg } for all examples here, it's required that table wrapper have background according to the selected theme